person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  # Getters
+  attr_accessor :name, :age
+  # Setters
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    return false if @age < 18
+
+    true
+  end
```

Comment:
- [ ] **[OPTIONAL]:** There's a shorter way of achieving the same result in one line, since this method returns true if the age is greater than 18, you can use that as the condition:

```suggestion
  def of_age?
    @age >= 18


  end
```

Ruby doesn't need to return in some cases explicitly, and this condition is going to be evaluated every time the method is called, returning either `true` or `false`. 🤓 🔧